packages/ui/src/proposals/model/proposalDescriptions.ts

```diff
@@ -33,6 +33,7 @@ export const proposalDescriptions: ProposalDescriptions = {
     'Membership Workgroup Lead is automatically assigned invitations upon taking the role. This proposals is aimed on managing this invitations count.',
   setReferralCut:
     'Referrals, same as new membership invitations are incentivised by the platform. Referral cut entails the reward to the originator of invitation links which resulted in new memberships.',
+  updatePalletFrozenStatus: 'You have the flexibility to enable or disable CRT feature.',
```

Comment:
Based on: https://www.figma.com/file/oD3UTElt3yc4JJfPJv4dn3?node-id=3:17686&mode=design#610787926
```suggestion
  updatePalletFrozenStatus: 'Enabling and disabling various features on the blockchain level.',
```